pkg/prometheus/agent/statefulset.go

```diff
@@ -207,9 +220,11 @@ func makeStatefulSetSpec(
 		if err != nil {
 			return nil, err
 		}
+		configReloaderWebConfigFile = confArg.Value
 		promArgs = append(promArgs, confArg)
 		volumes = append(volumes, configVol...)
 		promVolumeMounts = append(promVolumeMounts, configMount...)
+		configReloaderVolumeMounts = append(configReloaderVolumeMounts, configMount...)
```

Comment:
We could make an argument that adding the flag in all cases introduces a risk of breaking users not deploying the latest config-reloader tag. It would be safer to add the flag only when the web field isn't nil and we can remove the if condition in a few releases.

```suggestion
		// To avoid breaking users deploying an old version of the config-reloader image.
		// TODO: remove the if condition after v0.72.0.
		if cpf.Web != nil {
			configReloaderWebConfigFile = confArg.Value
			configReloaderVolumeMounts = append(configReloaderVolumeMounts, configMount...)
		}
```

Reply:
Thanks for your review. 

Got it. But `cpf.Web != nil` only determines whether `Prometheus's web` supports it, not whether the `config-reloader` supports it.

```
		if c.ReloaderConfig.Image.Version >= 0.69.0 {
		     // xxx
		}
```
Because config-reloader v0.69.0 already supports this feature.

Reply:
OR Add flag for Prometheus/alertmanager CRD?

Reply:
My intention was only to avoid breaking users that don't specify web config for their Prometheus/Alertmanager. For others, they still need to make sure that they use a recent-enough config-reloader image.

Reply:
@simonpasquier Add config-reloader image version check. PTAL